lib/modules/manager/hermit/index.ts

```diff
@@ -0,0 +1,14 @@
+import { HermitDatasource } from '../../datasource/hermit';
+import { id as versionId } from '../../versioning/hermit';
+export { updateArtifacts } from './artifacts';
+import { defaultConfig as partialDefaultConfig } from './default-config';
+export { extractPackageFile } from './extract';
+export { updateDependency } from './update';
+
+export const defaultConfig = {
+  fileMatch: partialDefaultConfig.fileMatch,
+  excludeCommitPaths: partialDefaultConfig.excludeCommitPaths,
```

Comment:
is `excludeCommitPath` a valid renovate option/ interface property? if not, remove it here